tests/pendulum_tests/test_timezone.py

```diff
@@ -36,3 +40,8 @@ def test_astimezone(self):
         d = d.astimezone('Europe/Paris')
         self.assertEqual('Europe/Paris', d.timezone_name)
         self.assertPendulum(d, now.year, now.month, now.day, now.hour + 1, now.minute)
+
+        if sys.version_info >= (3, 2):
+            d = d.astimezone(timezone.utc)
```

Comment:
I think it would be better to have a `timezone` instance that is not `utc` to catch any error that might happen in offset computation.